pkg/api/wrappers/deployment_install_client.go

```diff
@@ -25,6 +26,7 @@ type InstallStrategyDeploymentInterface interface {
 	DeleteDeployment(name string) error
 	GetServiceAccountByName(serviceAccountName string) (*corev1.ServiceAccount, error)
 	FindAnyDeploymentsMatchingNames(depNames []string) ([]*appsv1.Deployment, error)
```

Comment:
Can we remove `FindAnyDeploymentsMatchingNames`?

Reply:
We do use `FindAnyDeploymentsMatchingNames` on `checkForDeployments`.

Reply:
@ecordell @dinhxuanvu I'm tempted to say we should be indexing the deployment cache by the labels we care about.